srv/components/cortx_utils/init.sls

```diff
@@ -19,4 +19,6 @@ include:
   - .install
   - .post_install
   - .config
+{% if grains['id'] == 'srvnode-1' %}
```

Comment:
will checking primary node will help here ? 

`{% if "primary" in pillar["cluster"][grains["id"]]["roles"] %}`

Reply:
The suggestion from Foundation was to run it on any node, not just primary.. so kept srvnode-1 at present, will have to find a way to run it on all nodes one after another later on.